File: kotlin/src/main/kotlin/sammancoaching/MidLevel.kt
package sammancoaching

class MidLevel : SeniorityLevel {
    override fun getPensionContributionBonus(databaseAccess: SalaryContributionPercentages): Double {
        return databaseAccess.lookupValue(SalaryContributionPercentages.MID_SENIORITY_PERCENTAGE)
    }
}

File: php/src/MidLevel.php
<?php

declare(strict_types=1);

namespace PensionContribution;

final class MidLevel implements SeniorityLevelInterface
{
    public function getPensionContributionBonus(SalaryContributionPercentages $databaseAccess): float
    {
        return $databaseAccess->lookupValue(SalaryContributionPercentages::MID_SENIORITY_PERCENTAGE);
    }
}
